app/src/lib/stores/app-store.ts

```diff
@@ -3083,21 +3118,19 @@ export class AppStore extends TypedBaseStore<IAppState> {
     remote: IRemote
   ): PullRequest | null {
     const upstream = branch.upstreamWithoutRemote
-    if (!upstream) {
+
+    if (upstream == null) {
       return null
     }
 
-    for (const pr of pullRequests) {
-      if (
+    const pr = pullRequests.find(
+      pr =>
         pr.head.ref === upstream &&
-        pr.head.gitHubRepository &&
+        pr.head.gitHubRepository != null &&
         pr.head.gitHubRepository.cloneURL === remote.url
-      ) {
-        return pr
-      }
-    }
+    )
 
-    return null
+    return pr || null
```

Comment:
:lipstick: move the `|| null` up after the `find` function, to be consistent with our other usages